web/src/components/dashboard/SidebarCardsList.tsx

```diff
@@ -99,12 +99,23 @@ export const SidebarCardsList = (props: Props): ReactElement => {
     }
   };
 
+  const openInNewTab = (url: string): void => {
+    window.open(url, "_blank", "noreferrer");
```

Comment:
[pebble/dust] everywhere else in the code where we do a new tab, we do `rel="noreferrer noopener"` - do we need "noopener" here?

Reply:
Yeah, you're right, its a security cross site scripting thing